proto/streamlit/proto/Block.proto

```diff
@@ -30,7 +30,7 @@ message Block {
   }
 
   message Horizontal {
-    bool unused = 2;
+    int32 total_columns = 2;
```

Comment:
Since we permit floats/doubles to be used as weights, let's make this a double instead of an int32.

(and *maaaybe* rename to "total_weight" instead? or "total_columns_weight"? idk)